addons/web/static/src/core/utils/html.js

```diff
@@ -286,3 +286,27 @@ export function setElementContent(element, content) {
         element.textContent = content;
     }
 }
+
+export function isMarkup(value) {
+    return value instanceof Markup;
+}
+
+/**
+ * Compares two objects or their string representations for equality.
+ * If one is a markup object and the other is not, they are considered unequal.
+ * @param {any} a
+ * @param {any} b
+ * @returns {boolean}
+ */
+export function isHtmlTextuallyEqual(a, b) {
```

Comment:
We probably don't want to define the function for now, as it's used only once for now.